x-pack/legacy/plugins/apm/public/utils/httpStatusCodeToColor.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { StringMap } from '../../typings/common';
+
+const {
+  euiColorDarkShade,
+  euiColorSecondary,
+  euiColorWarning,
+  euiColorDanger
+} = theme;
+
+const httpStatusCodeColors: StringMap<string> = {
+  1: euiColorDarkShade,
+  2: euiColorSecondary,
```

Comment:
For "2xx" what do you think about using this?
```js
import { darken } from "polished";
import { palettes } from "@elastic/eui/lib/services";
const color2xx = darken(0.2, palettes.euiPaletteForStatus.colors[0]);
```

Demo: https://codesandbox.io/s/http-status-colors-oxcpi

Reply:
If we're keen to have custom colors for HTTP status, wouldn't it be better to specifically declare them in their own variable instead of passing in the EUI colors, which might change over time. We can grab the hex colors and just create our own set, and set the 2xx to the custom color you grabbed ^

Reply:
I'm good with that too. It just felt a bit more okay to extend the palette like this.

Reply:
Why are we creating our own success color? Primary is fine I guess but if you want something more consistent with the rest of the UI why aren't we using `euiColorSuccess` (![#017d73](https://placehold.it/15/017d73/000000?text=+) #017d73)?

Reply:
`euiColorSuccess` and `euiColorSecondary` are more blue-teal than green. Especially in dark mode they look very alien compared to the yellow warning and red error colors we have ("successColor" is the custom color I'm suggesting)

**Light:**
![image](https://user-images.githubusercontent.com/209966/66761626-ab730580-eea4-11e9-9b08-0f604a3ec0a9.png)

**Dark:**
<img width="883" alt="Screen Shot 2019-10-14 at 17 02 09" src="https://user-images.githubusercontent.com/209966/66761728-d8bfb380-eea4-11e9-9dd7-a30fb671fc35.png">

Reply:
If we are worried about the custom color, using the default will also work:
```
const color2xx = palettes.euiPaletteForStatus.colors[0];
```